src/main/java/com/example/nasa/controller/NasaController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.nasa.controller;
+
+import com.example.nasa.service.NasaPictureService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/picture")
+public class NasaController {
+
+    private final NasaPictureService nasaPictureService;
+
+    @PostMapping(produces = MediaType.IMAGE_JPEG_VALUE)
```

Comment:
You do not need MediaType because return type is void, also according to task the path should end with /pictures/steal